src/java.sql/share/classes/java/sql/BatchUpdateException.java

```diff
@@ -55,7 +55,7 @@
  * <P>
  * A JDBC driver implementation should use
  * the constructor {@code BatchUpdateException(String reason, String SQLState,
- * int vendorCode, long []updateCounts,Throwable cause) } instead of
+ * int vendorCode, long []updateCounts, Throwable cause) } instead of
```

Comment:
While we are here, is it more normal to say "long[] updateCounts", not separating the [] from the type.
Similarly at line 81, 118, 151, 247, 277, 318, 354.

Reply:
I thought about it too, but then noticed how the position of `[]` mimics that of the respective method signatures in code.

Reply:
OK, so lines 264,  295, 329, 364, 431 are arguably wrong as well?  Separating the [] completely looks quite rare.
I'll leave it up to you. 8-)

Reply:
I think that can be a follow on clean up.